aziotctl/aziotctl-common/src/set_log_level.rs

```diff
@@ -0,0 +1,68 @@
+use std::fmt;
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::process::Command;
+use std::str::FromStr;
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::module_name_repetitions)]
+/// # Errors
+///
+/// Will return `Err` if the systemd folder for the service does not exist or the user does not have
+/// permission to read it.
+pub fn set_log_level(services: &[&ServiceDefinition], level: LogLevel) -> Result<(), io::Error> {
+    for ServiceDefinition {
+        service,
+        sockets: _,
+    } in services
+    {
+        write_log(service, level)?;
+    }
+    Command::new("systemctl").arg("daemon-reload").output()?;
+
+    println!("Set log level to {} for all services. Run the `system restart` command for the changes to take effect.");
+    Ok(())
+}
+
+fn write_log(service: &str, level: LogLevel) -> Result<(), io::Error> {
+    let filename = format!("/etc/systemd/system/{}.d/log-level.conf", service);
```

Comment:
You need to create the `/etc/systemd/system/{}.d` directory. It won't exist by default.